docs/source/library-user-guide/extending-operators.md

```diff
@@ -43,11 +42,9 @@ fn rewrite(
         Ok(Transformed::no(plan))
     }
 }
-```
 
-```rust,ignore
 // Converts a uwheel aggregate result to a TableScan with a MemTable as source
-fn agg_to_table_scan(result: f64, schema: SchemaRef) -> Result<LogicalPlan> {
+fn agg_to_table_scan(result: f64, schema: SchemaRef) -> Result {
```

Comment:
I'm very confused why these changes are being made, especially considering it makes the example invalid?